pkg/restore/restore.go

```diff
@@ -814,6 +823,36 @@ func hasControllerOwner(refs []metav1.OwnerReference) bool {
 	return false
 }
 
+// isCompleted returns whether or not an object is considered completed
```

Comment:
Godoc doesn't distinguish based on line endings - you'll want to add a period at the end of this sentence.